conctl/containerd.py

```diff
@@ -116,6 +116,14 @@ def delete(self, *container_ids) -> CompletedProcess:
             )
         except CalledProcessError:
             return killed
+        finally:
+            # lp:1932052 ensure snapshots are removed on delete
+            try:
+                self._exec(
+                    'snapshot', 'rm', *container_ids
+                )
+            except CalledProcessError:
+                pass
```

Comment:
I chose to `pass` here because snapshots should be removed along with their container pretty quickly and `snapshot rm $foo` will fail if `$foo` is already gone.